cmd/yggctl/generate.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/redhatinsights/yggdrasil"
+)
+
+func generateDataMessage(messageType yggdrasil.MessageType, responseTo string, directive string, bytes []byte, metadata map[string]string, version int) (*yggdrasil.Data, error) {
```

Comment:
`bytes` parameter name is very generic; WDYT about `content` or something similar?

Reply:
This is funny. I had the parameter's originally named "content" and changed them to "bytes". I actually don't like `bytes` as much. It's the name of a stdlib package and can cause symbol collision later on if these files import the `bytes` package. I'll change them to `content`.